src/ActionMenu/ActionMenu.examples.stories.tsx

```diff
@@ -251,29 +250,33 @@ export const MultipleSections = () => {
         <IconButton icon={KebabHorizontalIcon} aria-label="Open menu" />
       </ActionMenu.Anchor>
       <ActionMenu.Overlay width="small">
-        <ActionList>
-          <ActionList.Group title="Raw file content">
-            <ActionList.Item onSelect={() => alert('Workflows clicked')}>Download</ActionList.Item>
-            <ActionList.Divider />
-            <ActionList.Item onSelect={() => alert('Workflows clicked')}>Jump to line</ActionList.Item>
-            <ActionList.Item onSelect={() => alert('Workflows clicked')}>Find in file</ActionList.Item>
-            <ActionList.Divider />
-            <ActionList.Item onSelect={() => alert('Workflows clicked')}>Copy path</ActionList.Item>
-            <ActionList.Item onSelect={() => alert('Workflows clicked')}>Copy permalink</ActionList.Item>
-          </ActionList.Group>
+        <ActionList
```

Comment:
This to me is this biggest breaking change. Visually it looks quite different, and the inclusion of `sx` concerns me. If it needs new styling to handle groups, we should add CSS to the component itself. Do you want to pair on this?

| before | after |
| -- | -- |
| ![image](https://user-images.githubusercontent.com/18661030/223758601-068a788c-f1e1-4ec6-9239-0a613b6713e7.png) | ![image](https://user-images.githubusercontent.com/18661030/223758653-b60c8359-47e4-4db9-af61-f2d182045c06.png) |

This is why I was hoping to see the snapshots with a diff compared to your changes here so we have a way to accurately test before/after. I think since you updated the snaps we won't see a failure here.

Reply:
Yeah, it'd be good to pair on this.

Reply:
Those aren't showing up for me locally (the weird overlay bubble things). Not sure what's up there.

Reply:
Super weird 🤔 @langermank I think for getting rid of them we should be able to just revert the commit that added them in from CI and then continue as normal!